packages/autoloader/composer.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "automattic/autoloader",
+	"description": "An autoloader for Jetpack packages",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"version": "1.0.0",
```

Comment:
Do we need a specific version here, or can we rely on the tags instead (like we're planning to do for the other packages)

Reply:
I am not sure how to make it work without a version here. 
I run into error when I start with an empty vendor directory.

Reply:
That makes sense, but it's a bug indeed. #12480 resolves that - it should unblock this once it lands.

The problem is that Composer needs a version when with the stable minimum stability constraint. But in #12480 we suggest to lower the minimum stability constraint to `@dev`, as it'll unblock us in our goal to always use the local package in Jetpack (be it for development or for production).